azure-communication-ui/calling/src/main/res/values-ar-rSA/azure_communication_ui_calling_strings.xml

```diff
@@ -29,9 +29,9 @@
     <string name="azure_communication_ui_calling_setup_view_background_gradient_accessibility_label">إعداد التدرّج</string>
     <string name="azure_communication_ui_calling_button_switch_camera_accessibility_label">تبديل الكاميرا</string>
     <string name="azure_communication_ui_calling_call_setup_action_bar_title">إعداد</string>
-    <string name="azure_communication_ui_calling_call_setup_toolbar_title_announcement"> Title</string>
```

Comment:
Why do we have a space here? Is it needed? Does it affect UI?

Reply:
@Mohtasim  could you please double check here for toolbar strings?

Reply:
The space is intended. These are announcement substring strings. So a space allows the talkback to announce with a pause.